cmd/protoc-gen-crd/main.go

```diff
@@ -46,6 +52,11 @@ func extractParams(parameter string) map[string]string {
 func generate(request *plugin.CodeGeneratorRequest) (*plugin.CodeGeneratorResponse, error) {
 	includeDescription := true
 	enumAsIntOrString := false
+	type genMetadata struct {
+		shouldGen           bool
+		includeExperimental bool
```

Comment:
nit: remove/rename references to experimental for clarity. Experimental should equal standard only right?